rasa/core/policies/ensemble.py

```diff
@@ -119,59 +119,82 @@ def _check_priorities(self) -> None:
                     docs=DOCS_URL_POLICIES,
                 )
 
-    # TODO: This is a temporary implementation for the RulePolicy prototype.
-    # Policies should be able to decide themselves whether to use rules or stories
-    # for training.
+    def _policy_ensemble_contains_policy_with_rules_support(self) -> bool:
+        """Determine whether the policy ensemble contains at least one policy
```

Comment:
optional: I'd either add a full docstring (including return type) or no docstring

Reply:
👍 forgot to add a return type